src/perl/vdotest/VDOTest/BlockDevice01.pm

```diff
@@ -46,22 +46,22 @@ my @deviceTypes = qw(
   fua-vdo
   iscsi-raw
   linear-raw
+  lvmvdo-crypt-raw
+  lvmvdo-dory-raw
   lvmvdo-iscsi
+  lvmvdo-iscsi-linear
   lvmvdo-linear
   lvmvdo-loop
   lvmvdo-raw
-  thin-raw
+  lvmvdo-tracer-raw
+  thin-thinpool-lvmvdo
+  thin-thinpool-vdo
+  thin-thinpool-raw
   tracer-raw
   vdo-linear
   vdo-loop
   vdo-raw
   vdo-stripfua
-  lvmvdo-crypt-raw
-  lvmvdo-dory-raw
-  lvmvdo-iscsi-linear
-  lvmvdo-tracer-raw
-  thin-lvmvdo-thin
-  thin-vdo-thin
   tracer-corruptor-raw
```

Comment:
The logic of the code motion here evades me.  Two things:
* The old list was two-device stacks, followed by three-devices stacks, each section alphabetized. We could preserve that. Or, it looks like you were trying to go for strictly alphabetical, in which case "tracer-corrupter-raw" and several of the vdo-foo stack are misalphabetized. I could see an argument for either choice, just please be consistent.
* Some combinations for device stacks have gone away (such as thin-vdo-thin). Is here a logic to which ones you removed? And can you describe that logic somewhere, either in the PR description or in the commit message?

Reply:
redone

Reply:
Okay, looks mostly reasonable. But the now thin-thinpool cases have more devices in the stack. So thin-thinpool is as two-devices stack, in should be listed with the other two device stacks. Similarly, the other new cases are three-device stacks and should go with the three-device stacks.